src/domain/application/utils.ts

```diff
@@ -6,9 +6,16 @@ import { AlluStatus, AlluStatusStrings, Application } from './types/application'
  * Save application to Haitaton backend
  */
 export async function saveApplication(data: Application) {
+  // If there is no hankeTunnus when creating new cable report application
+  // call /hakemukset/luo-hanke endpoint, so that hanke is generated in the backend
+  const postUrl: string =
+    data.hankeTunnus === null && data.applicationType === 'CABLE_REPORT'
+      ? `/hakemukset/johtoselvitys`
```

Comment:
Is there a need for these back ticks as there is no ${}?

Reply:
Good point, there is not. Changed that.